assets/components/errorMessage/errorMessage.jsx

```diff
@@ -10,16 +10,24 @@ import Svg from 'components/svg/svg';
 
 type PropTypes = {
   message: string,
+  id?: string,
```

Comment:
I think this needs to be:

```jsx
id: ?string
```

if you want to make it nullable. Also, the prop itself doesn't have to be optional, if it's present in `defaultProps` as it is below, flow will now figure out that it's optional.